libs/react-helpers/src/lib/format/number.ts

```diff
@@ -50,3 +51,10 @@ export const formatNumberPercentage = (value: BigNumber, decimals?: number) => {
     typeof decimals === 'undefined' ? Math.max(value.dp(), 2) : decimals;
   return `${value.dp(decimalPlaces).toFormat(decimalPlaces)}%`;
 };
+
+export const convertEthersBigNum = (
```

Comment:
nitpick (non-blocking): I think I would prefer to see a type overload on addDecimal rather than a separate method